File: MBShopBE/Models/Category.cs

using System.Collections.Generic;

namespace MBShopBE.Models
{
    public class Category
    {
        public int Id { get; set; }
        public string Label { get; set; }
        public int GroupId { get; set; }
        public ICollection<Product> Products { get; set; }
    }
}

File: MBShopFE/src/app/static/models/Category.ts
import { Product } from "./Product";

export class Category{
    id?: Number;
    label : String;
    groupId : Number;
    products : Product[];

    constructor(id: Number,label: String,groupId: Number){
        this.id=id;
        this.groupId=groupId;
        this.label=label;
    }
}